lib/fastly_nsq/message_queue/listener.rb

```diff
@@ -26,14 +26,14 @@ def process_next_message
 
     private
 
+    attr_reader :channel, :topic
+
     def process_one_message
       message = consumer.pop
-      MessageProcessor.new(message.body).go
+      MessageProcessor.new(topic: topic,  message_body: message.body).go
```

Comment:
I know with keyword args the order is not important... but it mildly bothers me that this is backwards from how it is defined below...